src/backbone.syphon.typeregistry.js

```diff
@@ -17,7 +17,11 @@ _.extend(TypeRegistry.prototype, {
   // found for the specified type, the default is
   // returned.
   get: function(type){
-    return this.registeredTypes[type] || this.registeredTypes['default'];
+    if (this.registeredTypes.hasOwnProperty(type)){
```

Comment:
This will not work in IE8
https://msdn.microsoft.com/en-us/library/ie/328kyd6z%28v=vs.94%29.aspx